style/modules/testimonial.css

```diff
@@ -0,0 +1,83 @@
+/* Desktop */
+/* Background Box */
+
+.testimonial {
+    margin-top: 100px;
+    max-width: 610px;
+    background: white;
+    border: 0.5px solid black;
+}
+
+/* Testimonial Picture */
+
+.testimonial--pic {
+    position: relative;
+    width: 200px;
+    height: 200px;
+    left: 50%;
+    margin-left: -100px;
+    transform: translateY(-80px);
+    overflow: hidden;
+}
+
+.testimonial--pic > img {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    margin: auto;
+    border-radius: 50%;
+    top: -100%;
+    left: -100%;
+    right: -100%;
+    bottom: -100%;
+}
+
+/* Text Container */
+
+.testimonial--content {
+    margin-top: -60px;
+    padding-left: 35px;
+    padding-right: 35px;
+}
+
+/* [[ Text ]] */
+
+.testimonial--content > h1 {
```

Comment:
For the people's names, I think we'd be better off with something other than a heading tag. One clue is that the testimonials aren't really page sections. `span` is fine (but note that you'll have to add `display: block`).

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#Usage_notes

Also, with `.module > tag` targets -- the upside is fewer class names, but the downside is that in cases like this you now can't update the tag. So I recommend just naming them:

```css
.testimonial--name {
  ...
}

.testimonial--quote {
  ...
}
```

Even more importantly, it documents the code better -- notice how you don't even need the comment `/* [[ text ]] */` after that, because the names tell you that it's text (and *which* text as well, which you'd otherwise have to look at the HTML file to figure out).